torchci/lib/benchmark/compilerUtils.ts

```diff
@@ -102,10 +102,11 @@ export function computePassrate(
 
     // If the model pass accuracy check but fails the performance benchmark with an
     // 0 speedup, it should be counted as a failure. However, `pass_due_to_skip` is
-    // an exception and it's ok to have 0 speedup there
+    // an exception and it's ok to have 0 speedup there, also `export` is an exception
+    // because we only measure its pass rate but not speedup.
     if (
       (isPass(bucket, workflowId, suite, compiler, model, passingModels) &&
-        record.speedup !== 0.0) ||
+        (record.speedup !== 0.0 || compiler === "inductor_export")) ||
```

Comment:
Hmm, didn't help the pass rate display. Maybe I should check `compiler === "export"`? @huydhn 

https://hud.pytorch.org/benchmark/compilers?dashboard=torchinductor&startTime=Thu%2C%2015%20Aug%202024%2021%3A14%3A53%20GMT&stopTime=Thu%2C%2022%20Aug%202024%2021%3A14%3A53%20GMT&granularity=hour&suite=torchbench&mode=inference&dtype=bfloat16&deviceName=cuda%20(a100)&lBranch=desertfire/ci_1&lCommit=905b1def9db263b24396950067278a94af4f3a11&rBranch=desertfire/ci_1&rCommit=905b1def9db263b24396950067278a94af4f3a11

Reply:
Oh, I see https://github.com/pytorch/test-infra/pull/5588 is already in.  Thank you for the follow-up fix!